src/App.tsx

```diff
@@ -2,57 +2,30 @@ import React from 'react';
 import logo from './logo.svg';
 import { Counter } from './features/counter/Counter';
 import './App.css';
+import {
+  createBrowserRouter,
+  RouterProvider,
+  Route,
+  createRoutesFromElements,
+} from "react-router-dom";
+import WelcomePage from './components/WelcomePage';
+import PageOne from './components/PageOne';
+import PageTwo from './components/PageTwo';
+import MainLayout from './components/MainLayout';
+
+const router = createBrowserRouter(
```

Comment:
Please, move this code to a separate file to keep the App root component clear. You can create a folder Navigation and put this file there. Call this file Router, as an example.